### PR TITLE
Blog style QA

### PR DESCRIPTION
**Pull request by broskoski**
A few blog style things I noticed when putting up the latest post.
- Puts back the shadow on the iframe and does responsive sizing so there is never a scrollbar
- Shrinks the margin a bit for images and iframes
- Adds styling for `hr`s
- Adds a margin for `h2` (which we only use on channel collection posts)
- Adds back the title

### CODE REVIEW

File: apps/blog/stylesheets/show.styl [hunk 1]
```diff
@@ -42,6 +42,24 @@
       iframe
         display block
         max-width 100%
-        margin 4em auto
         vertical-align bottom
         border 1px solid colors-gray-border
+        margin 2em auto
+
+      iframe
+        box-shadow 4px 4px 4px colors-gray-background
```
Comment (dzucconi) [inline apps/blog/stylesheets/show.styl]: what in the what

File: apps/blog/stylesheets/show.styl [hunk 2]
```diff
@@ -42,6 +42,24 @@
...
+        padding 1px
+        +below(400px)
+          height 2000px !important
+        +below(380px)
+          height 1950px !important
+        +below(340px)
+          height 1700px !important
+
+      h2 
+        margin-top 4em
```
Comment (dzucconi) [inline apps/blog/stylesheets/show.styl]: collapsed-margins(4em)

File: apps/blog/stylesheets/show.styl (hunk 1 above)
Comment (broskoski) [inline apps/blog/stylesheets/show.styl]: https://www.w3schools.com/cssref/css3_pr_box-shadow.asp

File: apps/blog/stylesheets/show.styl [hunk 3]
```diff
@@ -42,6 +42,24 @@
...
+
+      hr
+        border 1px solid colors-gray-border
+        margin 3em 2em
```
Comment (dzucconi) [inline apps/blog/stylesheets/show.styl]:
```
height 1px
background-color colors-gray-border
border none
```

**Pull request merged**